scripts/exregional_run_prepstart.sh

```diff
@@ -241,6 +249,41 @@ else
     cp_vrfy ${bkpath}/${restart_prefix}fv_srf_wnd.res.tile1.nc    fv_srf_wnd.res.tile1.nc
     cp_vrfy ${bkpath}/${restart_prefix}phy_data.nc                phy_data.nc
     cp_vrfy ${fg_root}/${YYYYMMDDHHmInterv}/${fg_restart_dirname}/INPUT/gfs_ctrl.nc  gfs_ctrl.nc
+
+# do SST update for 01Z onto the restart sfc_data.nc
```

Comment:
Line 253, change to "# do SST update at ${update_SST}z on the restart sfc_data.nc"

Reply:
Changed to "# do SST update at ${update_SST}z for the restart sfc_data.nc"